docs/_includes/topbar.html

```diff
@@ -117,37 +138,12 @@
                         </a>
                     </li>
                 {% endunless %}
+            {% else %}
                 <li>
-                    <a href="#" class="s-topbar--item" title="Inbox">
-                        {% icon "Inbox" %}
-                        <span class="s-activity-indicator s-activity-indicator__danger">3</span>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="s-topbar--item" title="Achievements">
-                        {% icon "Achievements" %}
-                        <span class="s-activity-indicator s-activity-indicator__success">+10</span>
-                    </a>
+                    <a href="#" class="s-topbar--item s-topbar--item__unset s-btn s-btn__filled ws-nowrap">Log in</a>
+                    <a href="#" class="s-topbar--item s-topbar--item__unset s-btn s-btn__primary ws-nowrap ml4">Sign up</a>
```

Comment:
These should probably be two separate `<li>`s, right?

Reply:
Done. That's something I had considered, but then decided not to do (for arbitrary reasons). It works either way and (in hindsight) your way is "more correct", so I've reverted that back.